docs/output.md

```diff
@@ -64,6 +72,33 @@ kraken2 classifies the reads. The important files are `*.classifiedreads.txt`, `
 
 </details>
 
+### bbduk
+
+bbduk classifies the reads. The important files are `*.bbduk.log` and `ids/*.bbduk.txt`. `<sample>` can be replaced by `<sample>_longReads`, `<sample>_R1` or left as `<sample>` depending on the cases mentioned in [fastp](#fastp).
```

Comment:
```suggestion
bbduk classifies the reads. The important files are `*.bbduk.log` and `ids/*.bbduk.txt`. `<sample>` can be replaced by `<sample>_longReads`, `<sample>_R1` or left as `<sample>` depending on the cases mentioned in [fastp](#fastp).
```

I find `bbduk classifies the reads` a bit vague... against what exactly and how?

Reply:
Also as general advice that I learnt from Phil - always put one sentence on a new line (markdown stitches them together unless there is an empty new line), as this makes diffs smaller and easier to see changes, e.g.:

```suggestion
bbduk classifies the reads.
The important files are `*.bbduk.log` and `ids/*.bbduk.txt`.
`<sample>` can be replaced by `<sample>_longReads`, `<sample>_R1` or left as `<sample>` depending on the cases mentioned in [fastp](#fastp).
```